deps/rabbitmq_prometheus/metrics-detailed.md

```diff
@@ -236,3 +236,33 @@ Group `channel_queue_exchange_metrics`:
 | Metric                                           | Description                                  |
 |--------------------------------------------------|----------------------------------------------|
 | rabbitmq_detailed_queue_messages_published_total | Total number of messages published to queues |
+
+### Virtual hosts and exchange metrics
+
+These can make sense in some scenarios, e.g. when vhost/exchanges are
+created using self-service automation. They are also a bit different
+from the rest of the metrics, as they not exactly per-node metrics,
+but cluster wide. So any aggregations of these values accross multiple
```

Comment:
We should instead say that aggregation of these metrics **must not** be done and will confuse the reader

Reply:
I was mostly thinking about sum/average. "Average number of (running) vhosts across all nodes" or "Sum of exchange numbers across all nodes" have no meaning. But min/max can be used as the easiest way to discard duplicate information. I'll think how to word this differently.